spec/spec.md

```diff
@@ -61,16 +66,19 @@ work is being done.
 
 ## Terminology
 
-Term | Definition
-:--- | :---------
-Decentralized Identifier (DID) | Unique ID string and PKI metadata document format for describing the cryptographic keys and other fundamental PKI values linked to a unique, user-controlled, self-sovereign identifier in a target system (i.e. blockchain, distributed ledger).
-Holder | The entity that submits proofs to a Verifier to satisfy the requirements described in a Presentation Definition
-Verifier | The entity that defines what proofs they require from a Holder (via a Presentation Definition) in order to proceed with an interaction.
+[[def:Decentralized Identifiers, Decentralized Identifiers, DID]]
+~ Unique ID URI string and PKI metadata document format for describing the cryptographic keys and other fundamental PKI values linked to a unique, user-controlled, self-sovereign identifier in a target system (i.e. blockchain, distributed ledger).
+
+[[def:Holder, Holders]]
+~ The entity that submits proofs to a [[ref:Verifier]] to satisfy the requirements described in a Presentation Definition
+
+[[def:Verifier, Verifiers]]
+~ The entity that defines what proofs they require from a [[ref:Holder]] (via a Presentation Definition) in order to proceed with an interaction.
```

Comment:
What presentations?

Reply:
Not sure, but I didn't change any of the definitions, I just put them into the format that will auto-link them. Let's address the contents of the terms in another PR. (I am sure they could use work, in addition incorporating all the other terms we identified). This was more to get the format in so people could party on adding/modding terms.